manim/mobject/value_tracker.py

```diff
@@ -71,7 +74,7 @@ def __init__(self, value=0, **kwargs):
         self.set_value(value)
 
     def get_value(self) -> float:
-        """Get the current value of the ValueTracker. This value changes continuously when :attr:`animate` for the ValueTracker is called."""
+        """Get the current value of this ValueTracker."""
         return self.points[0, 0]
 
     def set_value(self, value: Union[float, int]):
```

Comment:
Please change all the `Union[int, float]` simply to `float`.

```suggestion
    def set_value(self, value: float):
```